static/usage/v7/datetime/highlightedDates/array/index.md

```diff
@@ -0,0 +1,24 @@
+import Playground from '@site/src/components/global/Playground';
+
+import javascript from './javascript.md';
+import react from './react.md';
+import vue from './vue.md';
+
+import angular_example_component_html from './angular/example_component_html.md';
+import angular_example_component_ts from './angular/example_component_ts.md';
+
+<Playground
```

Comment:
The playground component currently defaults to version `6`. We will need to specify the explicit version on playground examples:

```tsx
<Playground version={7}
```

v6 playground examples will also need to be:
```tsx
<Playground version={6}
```

This flag is used to determine which stackblitz template files are used.